src/api/app/models/workflow/step/rebuild_package.rb

```diff
@@ -0,0 +1,44 @@
+class Workflow::Step::RebuildPackage < ::Workflow::Step
+  include Triggerable
```

Comment:
I really feel uncomfortable with using a controller concern in a model. Could the be something else?

Reply:
maybe because steps shouldn't be models? Another hint is having to call `Pundit.authorize` on that..

Reply:
I would prefer to to let this possible renaming of `Triggerable` controller concern to a different PR. Would that be ok @hennevogel ?

Reply:
sure thing, let's add a card to the backlog please

Reply:
> sure thing, let's add a card to the backlog please

Done